sovrin_client/test/cli/test_pool_upgrade_validation.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from sovrin_node.test.upgrade.conftest import validUpgrade
+from sovrin_client.test.cli.constants import INVALID_SYNTAX
+from plenum.common.util import randomString
+from sovrin_common.constants import JUSTIFICATION
+
+
+def testPoolUpgradeHasInvalidSyntaxIfJustificationIsEmpty(be, do, validUpgrade, trusteeCli):
+    validUpgrade[JUSTIFICATION] = ''
```

Comment:
It is not mandatory. It's just a validation for the case when we include justification. I mean you could have a typo or something else and submit an empty justification by mistake.